### DISCUSSION TITLE
SocketExeption in Unity

### DISCUSSION THREAD

**Discussion by ADVENCHER4** [consulo/UnityEditorConsuloPlugin#26] [General]
When I launch the project with an open IDE, Unity shows an error. If Consulo is not open, everything works fine
![Снимок экрана 2021-04-15 203112](https://user-images.githubusercontent.com/73076392/114913382-cd4aa100-9e29-11eb-9f20-86a8074d94bf.png)

**Comment by Game-Glide** [comment]
Are you using the unity consulo plugin? If yes, then this is a known UNFIXABLE bug. the only way to avoid it, is to disable "socket searching" in the consulo button in the toolbar

**Comment by VISTALL** [comment]
Hello. Can you show stacktrace? As said @YT-GameWorks socket searching can provide this bug (idk sometimes it can show this error, sometimes not). Can be ignored

Thanks

**Reply by ADVENCHER4** [reply]
Sorry, Idk how to show stacktrace

**Reply by VISTALL** [reply]
Click on message

**Reply by ADVENCHER4** [reply]
![Снимок экрана 2021-04-16 194549](https://user-images.githubusercontent.com/73076392/115057196-6d68fe80-9eec-11eb-8049-ee28d083f665.png)
This?

**Reply by VISTALL** [reply]
yeap. Interesting thing.
